packages/tracing/src/browser/metrics.ts

```diff
@@ -14,17 +14,6 @@ import { NavigatorDeviceMemory, NavigatorNetworkInformation } from './web-vitals
 
 const global = getGlobalObject<Window>();
 
-/**
- * Exports a way to add options to our metric collection. Currently experimental.
- */
-export interface MetricsInstrumentationOptions {
-  _reportAllChanges: boolean;
-}
-
-export const DEFAULT_METRICS_INSTR_OPTIONS: MetricsInstrumentationOptions = {
-  _reportAllChanges: false,
-};
```

Comment:
Removed this logic to save on bundle size. We can add back in an options object when we have more than 1 option.